pages/verification/index.tsx

```diff
@@ -157,14 +166,16 @@ export default function Verification(props: PageProps) {
                         <div className="pt-4 gap-y-4 flex flex-col h-min">
                             <text className="font-inter font-medium text-2xl text-justify">To send this message, please verify that you’re not a bot.</text>
                             <div className="border border-gray06 rounded-lg rounded-t-xl flex flex-col">
-                                <button className="shadow-3xl bg-humanId_blue" style={{
-                                    margin: '-1px',
-                                    backgroundImage: `url('/image/humanID_bg_button.svg')`,
-                                    backgroundSize: 'contain',
-                                    backgroundRepeat: 'no-repeat',
-                                    backgroundPosition: 'center',
-                                    height: '45px',
-                                    borderRadius: '8px'
+                                <button className="shadow-3xl bg-humanId_blue" 
+                                    disabled={isLoading}
+                                    style={{
+                                        margin: '-1px',
+                                        backgroundImage: `url('/image/humanID_bg_button.svg')`,
+                                        backgroundSize: 'contain',
+                                        backgroundRepeat: 'no-repeat',
+                                        backgroundPosition: 'center',
+                                        height: '45px',
+                                        borderRadius: '8px'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The button is now disabled based on the `isLoading` state, which is a good practice to prevent users from making multiple requests while one is still processing. However, it would be better to provide some visual feedback to the user when the button is disabled.

Consider changing the button's style or adding a spinner to indicate loading.
```diff
+                                     className={`shadow-3xl ${isLoading ? 'bg-gray-400' : 'bg-humanId_blue'}`}
                                      disabled={isLoading}
                                      style={{
                                          margin: '-1px',
                                          backgroundImage: `url('/image/humanID_bg_button.svg')`,
                                          backgroundSize: 'contain',
                                          backgroundRepeat: 'no-repeat',
                                          backgroundPosition: 'center',
                                          height: '45px',
                                          borderRadius: '8px'
                                      }} onClick={() => getLoginPage()} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->